reference/promise-types/files.markdown

```diff
@@ -681,6 +681,12 @@ Diffs for binary files are not generated. Files are considered binary files if [
 
 **Type:** `body copy_from`
 
+The copy_from body specifies the details for making remote copies.
+
+**Note:** Connections from the cf-agent to cf-serverd do not persist beyond a
+single agent run. Connections may be re-initialized for each bundle that uses a
+remote connection.
+
```

Comment:
Thanks Nick!

Personally I would try to mention it in a positive way, to present connection caching as a feature. "Connections from cf-agent to cf-serverd on the same host, get re-used for performance reasons. Implementation details currently restrict this connection caching only within the same bundle."